src/app/components/formular-gui/formular-gui.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
Komplett löschen, da keine spezifischen Tests gemacht werden. Ansonsten in den Testordner verschieben.